src/session_concurrency/request_handler.cc

```diff
@@ -170,9 +220,7 @@ RequestHandler::RecordResponse(
 
   request_record.response_timestamps_.push_back(end_time);
 
-  request_record.response_outputs_.emplace_back();
-
-  auto& response_outputs{request_record.response_outputs_.back()};
+  auto& response_outputs{request_record.response_outputs_.emplace_back()};
```

Comment:
A little cleanup. Since c++17, `emplace_back()` for vector actually returns a reference to the newly constructed.